draft_migration.sql

```diff
@@ -0,0 +1,61 @@
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+
+SET search_path = events_api, public;
+
+CREATE TABLE IF NOT EXISTS webhooks (
+                                        id BIGSERIAL NOT NULL,
+                                        condition TEXT NOT NULL,
+                                        flow_type TEXT NOT NULL,
+                                        event_type TEXT NOT NULL,
+                                        threshold_value FLOAT NOT NULL,
```

Comment:
we won't need `threshold_value` if we use the CEL language style condition, b/c it will just be part of the condition.